src/sass/variables/colors.scss

```diff
@@ -0,0 +1,2 @@
+$primary-color: #5993b8;
+$bg-color: #18191c;
```

Comment:
These are non-brand colors, we should probably try to stick with the established palette.